src/modules/Host.rb

```diff
@@ -57,12 +57,39 @@ def main
       # "hosts" file location
       @hosts_file = "/etc/hosts"
 
-      # Only write configuration
-      @write_only = false
-
       @initialized = false
     end
 
+    # Remove all entries from the host table.
+    def clear
+      @hosts = {}
+      modified = true
```

Comment:
I've obviously missed the modification mentioned by @mvidner above. I'll update the PR